cli_tools/common/utils/daisy/daisy_utils.go

```diff
@@ -82,6 +84,18 @@ var (
 	privacyTagRegex = regexp.MustCompile(`(\[Privacy\->)|(<\-Privacy\])`)
 )
 
+// OSChoices returns the supported OS identifiers, sorted.
+func OSChoices() []string {
+	choices := make([]string, len(osChoices))
+	i := 0
```

Comment:
Minor: you can avoid having a counter: https://yourbasic.org/golang/slice-of-keys-values-from-map/